src/Watchable.cpp

```diff
@@ -1,11 +1,10 @@
-#include "../include/Watchable.h"
-
+#include "../include/Session.h"
 
 
 Watchable::Watchable(long id, int length, const std::vector<std::string>& tags)
 : id(id), length(length), tags(tags){}
 
-std::string Watchable::getTagStrings()
+std::string Watchable::getTagStrings() // auxillary to avoid code duplication
```

Comment:
should be const as well.
in general, all get functions should be const

Reply:
Done